chess/frontend/gui/src/App.js

```diff
@@ -10,6 +11,7 @@ class App extends Component {
         <Switch>
           <Route path="/users" component={User} />
           <Route path="/sign_in" component={SignIn} />
+          <Route path="/dashboard" component={Dashboard} />
```

Comment:
By putting this line after the `"/sign_in"` line, does that make react call dashboard's `render()` after it calls sign_in's `render()`?